core/src/main/java/com/capitalone/dashboard/model/Issue.java

```diff
@@ -0,0 +1,187 @@
+package com.capitalone.dashboard.model;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * A specific commit in a version control repository.
+ *
+ * Possible collectors:
+ *  Subversion (in scope)
+ *  Git (in scope)
+ *  GitHub
+ *  TFS
+ *  BitBucket
+ *  Unfuddle
+ *
+ */
+@Document(collection="issues")
+public class Issue extends SCM {
+    @Id
+    private ObjectId id;
+    private ObjectId collectorItemId;
+
+    //Thus could be user id in GIT
+    private String orgName;
+    private String repoName;
+    private String number;
+    private String userId = null;
+    private String createdAt;
+    private String closedAt;
+    private long timestamp;
+    private long resolutiontime;
+
+    //These fields are for another collector to fill later - the ones
```

Comment:
If you wanted to ever show PR and ISSUE creations by developers, their departments, leadership etc, you'd want the PR and Issue to be associated with those fields. You'd expect another collector that can access a people data store and update these fields. However people datastores aren't a standard product across all enterprises unlike JIRA or git or subversion.